zad6/app/controllers/user_products_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UserProductsController < ApplicationController
+
+  def index
+    @products = Customer.find(params[:customer_id]).products
```

Comment:
Żeby posrtować produkty zgodnie z treścią zadania wystarczy dodać:
```ruby
 @products = Customer.find(params[:customer_id]).products.order(price: :desc)
```
Jeszcze lepiej byłoby dodać to jako scope do modelu `Product`